instance/iam.tf

```diff
@@ -1,11 +1,11 @@
 resource "aws_iam_instance_profile" "profile" {
-  count = "${var.instance_count == "0" ? 0 : 1}"
+  count = var.instance_count == 0 ? 0 : 1
```

Comment:
Isn't instance count always > 0? What's the use case for having it set to 0?

Reply:
The idea behind it is to be able to disable a specific instance in a specific environment but e can remove this option (not sure if it should be part of this PR)

Reply:
I agree that changing this should not be part of this PR